### PR TITLE
add db migration tool and run on start

### PR DESCRIPTION
**Pull request by blabadi**
Added migrate-mongo to migrate dbs

**Type of Change**

- [ ] Bug
- [ ] Refactor
- [x] New Feature

**Checklist before requesting review:**

- [ ] Check branch (code change PRs go to `develop` not master)
- [ ] Manual testing
- [ ] Regression tests completed and passing (double check number of tests).
- [ ] Spelling has been checked.
- [ ] Updated swagger docs accordingly (check it's still valid)

### CODE REVIEW

File: migrations/20200214223200-my-script.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  async up(db, client) {
+    await db.collection('newCollection').insertOne({ _id: '1', key: 'test' });
+    await db.collection('newCollection').deleteOne({ _id: '1', key: 'test' });
+    // you can write some tests to verify everything is good and throw error if not.
```
Comment (jaserud) [inline migrations/20200214223200-my-script.js]: Not critical, but it looks like `newCollection` stays in the db forever eh, do we wanna keep this collection?

**Comment by blabadi** [comment]
added @hlminh2000  since he used this tool before and recommended it

File: migrations/20200214223200-my-script.js (hunk 1 above)
Comment (blabadi) [reply migrations/20200214223200-my-script.js]: I can add a statement to drop it

Comment (jaserud) [reply migrations/20200214223200-my-script.js]: It's up to you, I won't loose sleep over it :stuck_out_tongue:

**Pull request merged**